MvvmCross.Forms/Platforms/Android/Core/MvxFormsAndroidSetup.cs

```diff
@@ -69,10 +75,18 @@ public virtual Application FormsApplication
 
         protected abstract Application CreateFormsApplication();
 
+        protected virtual IMvxFormsPagePresenter CreateFormsPagePresenter(IMvxFormsViewPresenter viewPresenter)
```

Comment:
Do we need to remove the code in the MvxFormsAndroidViewPresenter that creates the MvxFormsPagePresenter?